tests/python/frontend/tflite/test_forward.py

```diff
@@ -710,16 +710,23 @@ def test_forward_zeros_like():
 # Reduce
 # ------
 
-def _test_reduce(math_op, data, keep_dims=None):
+def _test_reduce(math_op, data, keep_dims=None, quantized=False):
```

Comment:
Suggest breaking into two functions. Another is _test_reduce_quantize

Reply:
> Suggest breaking into two functions. Another is _test_reduce_quantize

Done